src/lib/support/TimeUtils.cpp

```diff
@@ -464,6 +464,21 @@ bool CalendarTimeToSecondsSinceEpoch(uint16_t year, uint8_t month, uint8_t dayOf
     return true;
 }
 
+static void SecondsSinceEpochToCalendarTime(uint64_t secondsSinceEpoch, uint16_t & year, uint8_t & month, uint8_t & dayOfMonth,
+                                            uint8_t & hour, uint8_t & minute, uint8_t & second)
+{
+    uint32_t daysSinceEpoch = static_cast<uint32_t>(secondsSinceEpoch / kSecondsPerDay);
```

Comment:
This can have bad results if `secondsSinceEpoch` is large enough... but it never is in this file, because of the way this function is called.  This not-captured-by-the-type-system constraint should be documented, at the very least, as part of the API documentation for this function.